Dockerfile

```diff
@@ -1,34 +1,36 @@
 # Simple usage with a mounted data directory:
 # > docker build -t gaia .
-# > docker run -v $HOME/.gaiad:/root/.gaiad gaia init
-# > docker run -v $HOME/.gaiad:/root/.gaiad gaia start
-
-FROM alpine:edge
+# > docker run -it -p 46657:46657 -p 46656:46656 -v ~/.gaiad:/root/.gaiad -v ~/.gaiacli:/root/.gaiacli gaia gaiad init
+# > docker run -it -p 46657:46657 -p 46656:46656 -v ~/.gaiad:/root/.gaiad -v ~/.gaiacli:/root/.gaiacli jackzampolin/gaiad gaiad start
```

Comment:
We should reference `gaia`, not `jackzampolin/gaiad`; also, maybe this PR can address https://github.com/cosmos/cosmos-sdk/issues/1247?

Reply:
🤦‍♂️ Sorry about leaving that in. Meant to scrub that. Was planning to use this to work on #1247